### PR TITLE
LPD-32929 Information template mapping lost when publishing to live

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPD-32929

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/template/template-service/src/main/java/com/liferay/template/internal/exportimport/staged/model/repository/TemplateEntryStagedModelRepository.java [hunk 1]
```diff
@@ -143,8 +143,15 @@ public TemplateEntry updateStagedModel(
 			PortletDataContext portletDataContext, TemplateEntry templateEntry)
 		throws PortalException {
 
+		TemplateEntry existingTemplateEntry =
```
Comment (ealonso) [inline modules/apps/template/template-service/src/main/java/com/liferay/template/internal/exportimport/staged/model/repository/TemplateEntryStagedModelRepository.java:146]: @lfbesada This logic should be added in **TemplateEntryStagedModelDataHandler** when we call

**importedTemplateEntry = _stagedModelRepository.updateStagedModel(**

File: modules/apps/template/template-service/src/main/java/com/liferay/template/internal/exportimport/data/handler/TemplateEntryStagedModelDataHandler.java [hunk 2]
```diff
@@ -133,6 +137,9 @@ protected void doImportStagedModel(
 
 		importedTemplateEntry.setGroupId(portletDataContext.getScopeGroupId());
```
Comment (ealonso) [inline modules/apps/template/template-service/src/main/java/com/liferay/template/internal/exportimport/data/handler/TemplateEntryStagedModelDataHandler.java:139]: @lfbesada this empty line is not needed

File: modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/exportimport/content/processor/EditableValuesMappingExportImportContentProcessor.java [hunk 3]
```diff
@@ -220,6 +220,10 @@ private void _replaceMappedFieldExportContentReferences(
 		_exportTemplateReference(
 			portletDataContext, stagedModel, editableJSONObject);
 
+		if ((classNameId == 0) || (classPK == 0)) {
```
Comment (ealonso) [inline modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/exportimport/content/processor/EditableValuesMappingExportImportContentProcessor.java]: @lfbesada I think the commit message is wrong

**Comment by lfbesada** [comment]
@ealonso Thank you very much for your review. I have added your feedback, could you please take a look? Thank you :heart:

**Comment by lfbesada** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**